src/main/java/data/SheetsServiceUtil.java

```diff
@@ -31,9 +32,11 @@
 /** A class to create a Sheets instance. */
 public class SheetsServiceUtil {
 
-  public static Sheets getSheetsService() throws IOException, GeneralSecurityException {
-    // TODO(cxfloeder): Create a spreadsheet by using the OAuth tokens.
-
-    return null;
+  // Create a spreadsheet by using the OAuth access token.
```

Comment:
Comment is a bit misleading. Looks like you're creating a sheet, but really you're creating a connection to the Sheets service